src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/ExponentialMovingAverageCounter.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.Channel.Implementation
+{
+    using System;
+    using System.Threading;
+
+    /// <summary>
+    /// Exponential moving average counter.
+    /// </summary>
+    internal class ExponentialMovingAverageCounter
+    {
+        /// <summary>
+        /// Exponential coefficient (must be between 0 and 1).
+        /// </summary>
+        private readonly double coefficient;
```

Comment:
It is called "coefficient" in math literature, generally. Including Wikipedia article I referred to in the docs. So, I'll stick to "coefficient".